com.unity.multiplayer.mlapi/THIRD_PARTY_NOTICES.md

```diff
@@ -1,6 +1,17 @@
-MIT License
```

Comment:
This file should be called "Third Party notices.md" I think.

Reply:
Renamed.  It seemed weird for its case / spacing not to match the others, but then I found in the starter pack that this is indeed the standard way